### ISSUE TITLE
[2.0.24 release & stack profile] storage incremental blob copy on 2017-03-09-profile failure

### ISSUE THREAD

**Issue by derekbekoe** [Azure/azure-cli#5236]
```
$ docker run -it azuresdk/azure-cli-python:2.0.24
bash-4.3# az cloud update --profile 2017-03-09-profile 
bash-4.3# az storage blob incremental-copy start -h
The operation 'azure.multiapi.storage.v2015_04_05.blob.pageblobservice#PageBlobService.incremental_copy_blob' is invalid.
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/azure/cli/core/__init__.py", line 377, in get_op_handler
    op = getattr(op, part)
AttributeError: type object 'PageBlobService' has no attribute 'incremental_copy_blob'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/knack/cli.py", line 193, in invoke
    cmd_result = self.invocation.execute(args)
  File "/usr/local/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 241, in execute
    self.commands_loader.load_arguments(command)
  File "/usr/local/lib/python3.6/site-packages/azure/cli/core/__init__.py", line 201, in load_arguments
    self.command_table[command].load_arguments()  # this loads the arguments via reflection
  File "/usr/local/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 142, in load_arguments
    super(AzCliCommand, self).load_arguments()
  File "/usr/local/lib/python3.6/site-packages/knack/commands.py", line 76, in load_arguments
    cmd_args = self.arguments_loader()
  File "/usr/local/lib/python3.6/site-packages/azure/cli/core/__init__.py", line 332, in default_arguments_loader
    op = handler or self.get_op_handler(operation)
  File "/usr/local/lib/python3.6/site-packages/azure/cli/core/__init__.py", line 382, in get_op_handler
    raise ValueError("The operation '{}' is invalid.".format(operation))
ValueError: The operation 'azure.multiapi.storage.v2015_04_05.blob.pageblobservice#PageBlobService.incremental_copy_blob' is invalid.
bash-4.3# 
```
---

### Environment summary

Install Method (e.g. pip, interactive script, apt-get, Docker, MSI, edge build) / CLI version (`az --version`) / OS version / Shell Type (e.g. bash, cmd.exe, Bash on Windows)  

```
Docker / 2.0.24 / bash
```

**Comment by troydai** [comment]
Was this supported for the older profile?

**Comment by tjprescott** [comment]
I will fix this as part of #5239.
